Components/slider/slider.html

```diff
@@ -0,0 +1,137 @@
+                </li>
+                <li class="sidebar-link">
+                    <a class="sidebarlink" href="/Components/Grid/grid.html">Grid</a>
+                </li>
+                <li class="sidebar-link">
+                    <a class="sidebarlink" href="/Components/modal/modal.html">Modal</a>
+                </li>
+                <li class="sidebar-link">
+                    <a class="sidebarlink" href="/Components/Snackbar/snackbar.html">Snackbar</a>
+                </li>
+                <li class="sidebar-link active">
+                    <a class="sidebarlink" href="/Components/slider/slider.html">Slider</a>
+                </li>
+            </ul>
+        </nav>
+
+        <div class="page-container">
+            <h1 class="heading">Slider</h1>
+            <p class="info">Slider is used to take user input based on some range.</p>
+            <h1>Simple Slider</h1>
+            <div class="box">
+                <div class="slider">
+                    <div class="slider-range">
+                        <p>10</p>
+                        <p>50</p>
+                        <p>100</p>
+                    </div>
+                    <input type="range" name="slider" class="simple-slider" id="" min="10" max="100" value="50">
+                </div>
+            </div>
+            <div class="code-box">
+                <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=blackboard&wt=none&l=text%2Fx-csharp&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522slider%2522%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522slider-range%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%253E10%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%253E50%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%253E100%253C%252Fp%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cinput%2520type%253D%2522range%2522%2520name%253D%2522slider%2522%2520class%253D%2522simple-slider%2522%2520id%253D%2522%2522%2520min%253D%252210%2522%2520max%253D%2522100%2522%2520value%253D%252250%2522%253E%250A%253C%252Fdiv%253E%250A"
+                    style="width: 100%; height: 338px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
+
+            <h1>Styled Slider</h1>
+            <div class="box">
+                <div class="slider">
+                    <div class="slider-range">
+                        <p>10</p>
+                        <p>50</p>
+                        <p>100</p>
+                    </div>
+                    <input type="range" name="slider" class="styled-slider" id="" min="10" max="100" value="50">
```

Comment:
`id` field is empty, if it is not required then you can remove it

Reply:
I see same issue in other lines as well. Please check.

Reply:
thanks for pointing our i will make changes